src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -47,10 +50,35 @@
 #include "ui_automatedrssdownloader.h"
 #include "automatedrssdownloader.h"
 
+class AutomatedRssDownloader::DownloadRuleListMatchState
```

Comment:
OK, I'll probably respond tomorrow. Today, I am not in a state where I can read complex PRs.